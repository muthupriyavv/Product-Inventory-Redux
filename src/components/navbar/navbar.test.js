import React from 'react'
import Navbar from './navbar'
import {shallow} from 'enzyme';

describe('navbar renders' , () => {
    it('render navbar component correctly', () => {  
        const NavbarComponent = shallow(<Navbar></Navbar>);
        expect(NavbarComponent).toMatchSnapshot();
    });
})