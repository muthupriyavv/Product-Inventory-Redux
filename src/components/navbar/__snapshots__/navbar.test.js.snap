// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navbar renders render navbar component correctly 1`] = `ShallowWrapper {}`;
